requirements.txt

```diff
@@ -1,2 +1,8 @@
-Twisted==17.9
+certifi==2017.11.5
```

Comment:
Personally, I wouldn't pin the dependencies of the libraries that we use as I think that just adds more work to keep current.

Reply:
but repeatable builds!